Lesson 1 Concepts and Project

aprilgittens opened the pull request:
As a reminder, images still need to be added. However, I have placeholders in place. I haven't removed the hyperlinks yet as I wanted to focus on sharing what I have thus far.

aprilgittens committed:
Add what is MR

aprilgittens committed:
add devices and how to decide

aprilgittens committed:
add setup your pc for MR dev

aprilgittens committed:
add to get started with unity

aprilgittens committed:
add how to get started with MRTK and unity

Yonet commented on lessons/lesson1/concepts/how-do-i-decide-if-i-need-to-develop-for-virtual-reality-or-augmented-reality.md:
```diff
@@ -6,3 +6,25 @@
+
+Technological advancement is what has enabled mixed reality experiences. There are no devices today that can run experiences across the entire spectrum. However, Windows 10 provides a common mixed reality platform for both device manufacturers and developers. Devices today can support a specific range within the mixed reality spectrum. Over time, new devices will expand that range. In the future, ...
+
+Often, it is best to think what type of experience an application or game developer wants to create. The experiences will typically target a specific point or part on the spectrum. Then, developers should consider the capabilities of devices they want to target. For example, experiences that rely on the physical world will run best on HoloLens.
```
> For example, experiences that rely on the physical world will run best on HoloLens.
 Please rephrase to Augmented reality, instead of HoloLens. AR includes Android and other Mixed Reality devices. We want to cover at least the mobile developers in the curriculum.

Yonet commented on lessons/lesson1/concepts/how-do-i-decide-if-i-need-to-develop-for-virtual-reality-or-augmented-reality.md:
```diff
@@ -6,3 +6,25 @@
+|**Movement**     |    Full six-degrees-of-freedom movement, both rotation and translation.     |    Full six-degrees-of-freedom movement, both rotation and translation.     |
+
+
+Note, whether a device is connected to or tethered to a separate PC (via USB cable or Wi-Fi) or self-contained (untethered) does not reflect whether a device is holographic or immersive. Certainly, features that improve mobility lead to better experiences, and both holographic and immersive devices could be tethered or untethered.
```
Above section, although useful, a little distracting. You could have it as another question for example.

Yonet commented on lessons/lesson1/concepts/how-do-i-decide-if-i-need-to-develop-for-virtual-reality-or-augmented-reality.md:
```diff
@@ -6,3 +6,25 @@
+
+- **Near physical reality**. Users remain present in their physical environment and are never made to believe they have left that environment.
+- **Fully mixed reality**. These experiences blend the real world and the digital world. Viewers who have seen the movie [Jumanji](https://en.wikipedia.org/wiki/Jumanji) can reconcile how the physical structure of the house where the story took place was blended with a jungle environment.
+- **Near digital reality**. Users experience a completely digital environment, and are unaware of what occurs in the physical environment around them.
```
Please make the answer to which reality to choose more precisely and move the descriptions of the concepts into its own question.

Check out: What's the best reality for my app?

https://twitter.com/WindowsDocs/status/1212766878235602944

Yonet reviewed (commented):
Please follow the structure of questions and answers please, as shown in this example: https://github.com/Yonet/AzureMixedRealityDocs/tree/master/lessons/lesson-7/concepts.

Thanks!

aprilgittens committed:
remove device info

aprilgittens committed:
create what device should i use question

aprilgittens committed:
replace hololens with AR

aprilgittens committed:
remove which reality section

aprilgittens committed:
add what is the best reality for my app question

aprilgittens committed:
Merge remote-tracking branch 'upstream/master' into lesson1concepts

aprilgittens committed:
update text

aprilgittens committed:
add image mr spectrum image

aprilgittens committed:
add images for best reality for app

aprilgittens committed:
add images to pc setup

aprilgittens committed:
add images to get started with unity

aprilgittens committed:
add images to mr dev with unity

aprilgittens committed:
change titles to lowercase

aprilgittens commented:
@Yonet I just changed the titles to lowercase for lesson 1. Also, I had actually already changed the form while you were away for this lesson. Please review and let me know if what I've added is still not in the proper format.
